### PR TITLE
Build: space between curly and paren is optional

### PR DESCRIPTION
**Pull request by timmywil**
Fixes gh-2399

### CODE REVIEW

File: build/tasks/build.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ module.exports = function( grunt ) {
 	var fs = require( "fs" ),
 		requirejs = require( "requirejs" ),
 		srcFolder = __dirname + "/../../src/",
-		rdefineEnd = /\}\);[^}\w]*$/,
+		rdefineEnd = /\}\s*?\);[^}\w]*$/,
```
Comment (mzgol) [inline build/tasks/build.js]: A very minor nit: the non-greediness doesn't matter here as `\)` doesn't match `\s` anyway, other regexps in this file don't use it.

Comment (timmywil) [inline build/tasks/build.js]: I generally use non-greedy unless I explicitly want greediness.

Comment (mzgol) [inline build/tasks/build.js]: OK, doesn't matter.

**Comment by mzgol** [comment]
LGTM.

**Pull request closed without merging**